flask/app/routes.py

```diff
@@ -83,9 +83,9 @@ def login():
 @routes.route("/api/logout", methods=["POST"])
 @login_required
 def logout():
-    if not request.json:
-        app.logger.error("Logout failed as request body is not JSON")
-        abort(400, description="Request body must be JSON!")
+    if request.headers.get("Content-Type") != "application/json":
+        app.logger.error("Logout failed as Content-Type is not JSON")
+        abort(415, description="Content-Type must be JSON!")
```

Comment:
What's the reason for not using the decorator here?

Reply:
oops! fixed in  [75f8c38](https://github.com/ccmbioinfo/stager/pull/484/commits/75f8c38e245b8d2ffed64bd849c9df2c948638af)